docs/source/index.rst

```diff
@@ -17,6 +18,8 @@ Augur Documentation
    schema/toc
    login
    procedures/toc
+   migrations/toc
```

Comment:
You added a reference to a file that doesnt exist - this is causing a warning in the docs build (and presumably causing it to fail)

Might be nice to add a toc page (similar to the other TOC pages) to act as both a Table of contents for the migrations pages and also a general page explaining what the migrations sections of the docs are for

Reply:
`/home/runner/work/augur/augur/docs/source/index.rst:7: WARNING: toctree contains reference to nonexisting document 'migrations/toc'`